# this file is read into a `SemanticCloud::Settings` struct and used withinSemanticCloud

# +x: forward, +y: left
cloud:
  max_point_x: 20
  min_point_x: -5
  max_point_y: 10
  min_point_y: -10

bev:
  image_rows: 500 # to be subsampled to 256
  image_cols: 400 # to be subsampled to 205 (204.8)

# max leaf
kd_max_leaf: 128

mask:
  # squared distance of the closest point to the pixel center (higher = more masking)
  stitching_threshold: 0.2
  # number of neighbors to consider in the kd lookup
  knn_count: 16
  # "padding" of the search region for the egovehicle mask
  vehicle_mask_padding: 5